src/qt/bitcoingui.cpp

```diff
@@ -516,13 +516,13 @@ void BitcoinGUI::setClientModel(ClientModel *_clientModel)
         }
 #endif // ENABLE_WALLET
         unitDisplayControl->setOptionsModel(_clientModel->getOptionsModel());
-        
+
```

Comment:
nit, don't add unrelated changes just because your editor changed it, same below.

Reply:
I put back the spaces, and I'm getting `lint` error from Travis:

https://travis-ci.org/bitcoin/bitcoin/jobs/384047372#L756

```
@@ -525 +525 @@ void BitcoinGUI::setClientModel(ClientModel *_clientModel)
+
^---- failure generated from contrib/devtools/lint-whitespace.sh
```